src/yesql/generate.clj

```diff
@@ -86,11 +86,13 @@
 (tc-ignore
  (defn query-handler
    [db sql-and-params
-    {:keys [row-fn result-set-fn]
-     :or {row-fn identity
+    {:keys [row-fn result-set-fn identifiers]
+     :or {identifiers s/lower-case
```

Comment:
As per the projects codings standards (which only exist in my head), can you either qualify this as `string/lower-case`, or just refer in `lower-case`. I don't mind which, but I'd rather not have `s` as a namespace. :-)